loader/validate.go

```diff
@@ -38,6 +38,12 @@ func checkConsistency(project *types.Project) error {
 			}
 		}
 
+		for dependedService := range s.DependsOn {
```

Comment:
Threw in a freebie, eh? Normally I'd say separate PR but it's a small change so 👍

Reply:
Wow! Good catch! But I didn't add that 😮 
It's already on `master` https://github.com/compose-spec/compose-go/blob/master/loader/validate.go#L41-L45

Reply:
I've just rebased and it seems to remove this section from the diff